src/locale/he.json

```diff
@@ -185,5 +186,25 @@
   "plannedRoute": "מסלול מתוכנן",
   "actualRoute": "מסלול בפועל",
   "youtube_modal_info_title": "סרטון הסבר",
-  "open_video_about_this_page": "לפתוח סרטון על העמוד הזה"
+  "open_video_about_this_page": "לפתוח סרטון על העמוד הזה",
+  "publicAppealPage": {
+    "title": "קול קורא",
+    "tasks": [
```

Comment:
Can you please copy and paste the exact same text to the `en.json` file?

Reply:
done